spring-pulsar-docs/src/main/antora/modules/ROOT/pages/reference/pulsar.adoc

```diff
@@ -173,17 +173,16 @@ public void listen(String message) {
 }
 ----
 
-In this most basic form, you must provide the following two properties with their corresponding values:
+In this most basic form, you must still provide the topic name by setting the following property:
 
 [source,yaml,indent=0,subs="verbatim"]
 ----
-spring.pulsar.consumer:
-  topics: hello-pulsar
-  subscription-name: hello-pulsar-subscription
+spring.pulsar.consumer.topics: hello-pulsar-topic
 ----
 
-NOTE: If the topic information is not directly provided, a <<topic-resolution-process-imperative,topic resolution process>> is used to determine the destination topic.
+When `subscription-name` is not provided an auto-generated subscription name will be used.
```

Comment:
Same typo as above.

Perhaps also worth being more explicit and saying _not provided on `@PulsarListener` annotation_?